explorer/dockerExplorer.ts

```diff
@@ -0,0 +1,163 @@
+import * as vscode from 'vscode';
+import * as fs from 'fs';
+import * as path from 'path';
+import { docker } from '../commands/utils/docker-endpoint';
+
+export class DockerExplorerProvider implements vscode.TreeDataProvider<DockerNode> {
+
+    private _onDidChangeTreeData: vscode.EventEmitter<DockerNode | undefined> = new vscode.EventEmitter<DockerNode | undefined>();
+    readonly onDidChangeTreeData: vscode.Event<DockerNode | undefined> = this._onDidChangeTreeData.event;
+    private _imagesNode: DockerNode;
+    private _containersNode: DockerNode;
+    private _registriesNode: DockerNode;
+    private _debounceTimer: NodeJS.Timer;
+    
+    constructor() {
```

Comment:
`_registriesNode`? I commented it out to match what i did with other "registry" nodes.